packages/react-component-library/src/components/Button/index.tsx

```diff
@@ -1,43 +1 @@
-import React from 'react'
```

Comment:
merge this commit with earlier commit for button export change and the button.tsx should be in there too